### DISCUSSION TITLE
Tailwind does not work after fresh installation

### DISCUSSION THREAD

**Discussion by harryhanYuhao** [tailwindlabs/tailwindcss#12678] [Help] [answered]
**What version of Tailwind CSS are you using?**

v3.4.0 

**What build tool (or framework if it abstracts the build tool) are you using?**

tailwindcss 

**What version of Node.js are you using?**

v18.19

**What browser are you using?**

Firefox

**What operating system are you using?**

Fedora ''

**Reproduction URL**

I am following exactly what is shown on the getting started [guide](https://tailwindcss.com/docs/installation). 

**Describe your issue**

Tailwind css simply does not work. It seems like other people are also facing similar problems as shown here: https://stackoverflow.com/questions/70577297/tailwind-class-is-not-working-after-installed

Tailwind works after changing 

```
/* input.css */
@tailwind base;

@tailwind components;

@tailwind utilities;
```

to 
```
@import "tailwindcss/base";

@import "tailwindcss/components";

@import "tailwindcss/utilities";
```

**Comment by adamwathan** [comment, marked as answer]
Hey! Can you create a GitHub repository that demonstrates things not working? Can definitely find the issue in your setup if you can share the actual code.

**Reply by harryhanYuhao** [reply]
Thank you for your reply. Here is the repo: https://github.com/harryhanYuhao/tailwindQuickStart

I am running the command `npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch` in the root directory. 

I ran the server with `python -m http.server -d src`

**Reply by harryhanYuhao** [reply]
I think the problem is `python -m http.server -d src`. This make the server unable to find `/dist/output.css`. 

By running`python -m http.server` it is working.
